codegen/tests/ui/next/invalid-endpoint-syntax.stderr

```diff
@@ -0,0 +1,23 @@
+error: expected `,`
+
+         = help: The endpoint takes a comma-separated list of keys and values: `endpoint(method = get, route = "/hello")`
```

Comment:
I think we can drop this `help` as it might confuse users a bit

Reply:
Maybe replace it with `The endpoint takes two arguments: `endpoint(method = get, route = "/hello")``, or would you prefer an empty  `= help:`?

Reply:
I tried the above but it was still a bit confusing, I'll leave it blank unless we get a good idea for this message.